mParticle-Rokt/MPKitRokt.m

```diff
@@ -52,6 +61,29 @@ - (void)start {
 
     dispatch_once(&kitPredicate, ^{
         self->_started = YES;
+        
+        NSDictionary *attributes = @{
+                @"email" : @"j.smith@example.com",
+                @"firstname": @"Jenny",
+                @"lastname": @"Smith",
+                @"mobile": @"(555)867-5309",
+                @"postcode": @"90210",
+                @"country": @"US"
+            };
+            [Rokt executeWithViewName:@"RoktExperience"
+                           attributes:attributes
+                           placements:nil
+                               onLoad:^{
+                // Optional callback for when the Rokt placement loads
+            } onUnLoad:^{
+                // Optional callback for when the Rokt placement unloads
+            } onShouldShowLoadingIndicator:^{
+                // Optional callback to show a loading indicator
+            } onShouldHideLoadingIndicator:^{
+                // Optional callback to hide a loading indicator
+            } onEmbeddedSizeChange:^(NSString *selectedPlacement, CGFloat widgetHeight) {
+                // Does not required for Full screen overlay
+            }];
```

Comment:
This can be removed since these are hardcoded, vs the method below which is the actual mapping, right?

Reply:
My apologies. This was me testing something out and forgetting to remove it.